pydoctor/templatewriter/pages/__init__.py

```diff
@@ -45,10 +45,13 @@ def map_kind(kind: model.DocumentableKind) -> model.DocumentableKind:
 
     return (-o.privacyClass.value, -map_kind(o.kind).value if o.kind else 0, o.fullName().lower())
 
-def format_decorators(obj: Union[model.Function, model.Attribute]) -> Iterator["Flattenable"]:
+def format_decorators(obj: Union[model.Function, model.Attribute, model.FunctionOverload]) -> Iterator["Flattenable"]:
+    # Use the overload's primary function for some parts here
```

Comment:
Suggesting a more verbose comment here, thanks for the explanations.

```suggestion
    # Since we use this function to colorize the FunctionOverload decorators and it's not an actual Documentable subclass, we use the overload's 
    # primary function for parts that requires an interface to Documentable methods or attributes
```